saleor/graphql/order/types.py

```diff
@@ -194,7 +196,21 @@ def get_payment_status_for_order(order):
     return status
 
 
-class OrderGrantedRefund(ModelObjectType):
+class OrderGrantedRefundLine(ModelObjectType[models.OrderGrantedRefundLine]):
+    id = graphene.GlobalID(required=True)
+    quantity = graphene.Int(description="Number of items to refund.", required=True)
+    order_line = graphene.Field(
```

Comment:
Missing resolver with dataloader for this field.

Reply:
@fowczarek  Added :)